app/util.py

```diff
@@ -119,3 +126,27 @@ def project_user_permissions(form):
         permissions[user] = user_permissions
 
     return permissions
+
+
+# Padding for encryption
+# The lambdas are from StackOverflow, I have no idea how they work
+def pad(e):
+    # Block size
+    BS = 16
+    return (lambda s: s + (BS - len(s) % BS) * chr(BS - len(s) % BS))(e)
+
+
+def unpad(e):
+    return (lambda s: s[:-ord(s[len(s)-1:])])(e)
```

Comment:
Working from the inside out: Get the last character the string (e becomes s in the lambda). Get the ord() of that character - which from the last lambda should be the number of characters we padded the original string with. Then measure from the end of our string backwards the number of characters we added (which ord of the last character gave us). Get all the characters in the string up to that point. So if we added 10 chars, get all the chars in the string up to 10 from the end. Which leaves us with the original string.